policy/datasets.yaml

```diff
@@ -1246,6 +1246,18 @@ processStar_metadata:
     storage: YamlStorage
     python: lsst.daf.base.PropertySet
     template: processStar_metadata/%(visit)d_metadata.yaml
+spectractorOutputRoot:
+    description: "A dataset to allow butlers to get paths for Spectractor output"
+    persistable: ignored
+    python: None
+    storage: PickleStorage
+    template: 'spectractorOutput/v%(expId)d/'
+spectractorPlots:
```

Comment:
Should this also be a "Root", since it's a directory?  spectractorPlotRoot?

Reply:
Removed entirely 🙂